tests/appsec/waf/test_blocking.py

```diff
@@ -8,106 +9,27 @@
 if context.library == "cpp":
     pytestmark = pytest.mark.skip("not relevant")
 
-HTML_DATA = """<!-- Sorry, you’ve been blocked -->
-<!DOCTYPE html>
-<html lang="en">
-
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width,initial-scale=1">
-  <title>You've been blocked</title>
-  <style>
-    a,
-    body,
-    div,
-    html,
-    span {
-      margin: 0;
-      padding: 0;
-      border: 0;
-      font-size: 100%;
-      font: inherit;
-      vertical-align: baseline
-    }
-
-    body {
-      background: -webkit-radial-gradient(26% 19%, circle, #fff, #f4f7f9);
-      background: radial-gradient(circle at 26% 19%, #fff, #f4f7f9);
-      display: -webkit-box;
-      display: -ms-flexbox;
-      display: flex;
-      -webkit-box-pack: center;
-      -ms-flex-pack: center;
-      justify-content: center;
-      -webkit-box-align: center;
-      -ms-flex-align: center;
-      align-items: center;
-      -ms-flex-line-pack: center;
-      align-content: center;
-      width: 100%;
-      min-height: 100vh;
-      line-height: 1;
-      flex-direction: column
-    }
 
-    p {
-      display: block
-    }
+_CUR_DIR = os.path.dirname(os.path.abspath(__file__))
 
+# Initial template version as found in Java, with different indentation to v1.
+BLOCK_TEMPLATE_HTML_V0 = open(os.path.join(_CUR_DIR, "blocked.v0.html"), "r").read().rstrip()
```

Comment:
I don't think we should need a rstrip() everywhere here ? If we copy the exact file then by doing a rstrip() you run the risk to delete a character that was in the spec

Reply:
I'll try removing it, I think that we have some inconsistency about ending with new line or not across tracers (that's why the test had `rstrip()` before.

We could probably be more lenient about previous versions of the blocking template, and more strict for `test_html_template_v2`? One of the principles I followed is that whatever is done for previous versions of the template and was not checked in the system-tests before, it's done and we shouldn't tag for it. And the increased strictness should apply only to new checks, since it has a new `@released` decorator.

Reply:
I have removed all `rstrip` calls. Unless this fails for a language where this test is already enabled, I think it shouldn't be a problem.